routes/user.js

```diff
@@ -14,5 +14,6 @@ router.use('/:id/:role/reviews', reviewRouter)
 router.get('/', asyncWrapper(userController.getUsers))
 router.get('/:id/:role', asyncWrapper(userController.getOneUser))
 router.delete('/:id', asyncWrapper(userController.deleteUser))
+router.post('/:id/deactivation', asyncWrapper(userController.userDeactivation))
```

Comment:
read the task again
1) call it /delete-my-profile or so
2) use middleware that sets id and a role of a current user